src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XMLMessages.properties

```diff
@@ -128,15 +128,15 @@
         EncodingDeclRequired = The encoding declaration is required in the text declaration.
         NoMorePseudoAttributes = No more pseudo attributes are allowed.
         MorePseudoAttributes = More pseudo attributes are expected.
-        PseudoAttrNameExpected = A pseudo attribute name is expected. 
+        PseudoAttrNameExpected = A pseudo attribute name is expected.\u0020
 # 4.3.2 Well-Formed Parsed Entities
         CommentNotInOneEntity = The comment must be entirely contained within the same parsed entity.
         PINotInOneEntity = The processing instruction must be entirely contained within the same parsed entity.
 # 4.3.3 Character Encoding in Entities
         EncodingDeclInvalid = Invalid encoding name \"{0}\".
         EncodingByteOrderUnsupported = Given byte order for encoding \"{0}\" is not supported.
         InvalidByte = Invalid byte {0} of {1}-byte UTF-8 sequence.
-        ExpectedByte = Expected byte {0} of {1}-byte UTF-8 sequence.  
+        ExpectedByte = Expected byte {0} of {1}-byte UTF-8 sequence.\u0020\u0020
```

Comment:
Same here as well, either a mistake or should be ` ExpectedByte = Expected byte {0} of {1}-byte UTF-8 sequence.\u0020`